### ISSUE TITLE
uninitialized constant Telegrammer::DataTypes::Message::Voice

### ISSUE THREAD

**Issue by TheZ3ro** [mayoral/telegrammer#10]
Just updated fromtelegrammer 0.3.0 to 0.4.0 and this error shows up when `require  "telegrammer"`

```
/home/thezero/.rvm/rubies/ruby-2.0.0-p451/lib/ruby/gems/2.0.0/gems/virtus-1.0.5/lib/virtus/const_missing_extensions.rb:14:in `const_missing': uninitialized constant Telegrammer::DataTypes::Message::Voice (NameError)
    from /home/thezero/.rvm/rubies/ruby-2.0.0-p451/lib/ruby/gems/2.0.0/gems/telegrammer-0.4.0/lib/telegrammer/data_types/message.rb:44:in `<class:Message>'
    from /home/thezero/.rvm/rubies/ruby-2.0.0-p451/lib/ruby/gems/2.0.0/gems/telegrammer-0.4.0/lib/telegrammer/data_types/message.rb:30:in `<module:DataTypes>'
    from /home/thezero/.rvm/rubies/ruby-2.0.0-p451/lib/ruby/gems/2.0.0/gems/telegrammer-0.4.0/lib/telegrammer/data_types/message.rb:2:in `<module:Telegrammer>'
    from /home/thezero/.rvm/rubies/ruby-2.0.0-p451/lib/ruby/gems/2.0.0/gems/telegrammer-0.4.0/lib/telegrammer/data_types/message.rb:1:in `<top (required)>'
    from /home/thezero/.rvm/rubies/ruby-2.0.0-p451/lib/ruby/site_ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
    from /home/thezero/.rvm/rubies/ruby-2.0.0-p451/lib/ruby/site_ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
    from /home/thezero/.rvm/rubies/ruby-2.0.0-p451/lib/ruby/gems/2.0.0/gems/telegrammer-0.4.0/lib/telegrammer.rb:20:in `<top (required)>'
    from /home/thezero/.rvm/rubies/ruby-2.0.0-p451/lib/ruby/site_ruby/2.0.0/rubygems/core_ext/kernel_require.rb:135:in `require'
    from /home/thezero/.rvm/rubies/ruby-2.0.0-p451/lib/ruby/site_ruby/2.0.0/rubygems/core_ext/kernel_require.rb:135:in `rescue in require'
    from /home/thezero/.rvm/rubies/ruby-2.0.0-p451/lib/ruby/site_ruby/2.0.0/rubygems/core_ext/kernel_require.rb:144:in `require'
    from bin/telegrammer-plugin:7:in `<main>'
```

**Comment by mayoral** [comment]
Sorry, I forgot to add a require for that file. Fixed in 0.4.1, just uploaded.

**Comment by TheZ3ro** [comment]
Thanks :+1:
